ouroboros-consensus/src/Ouroboros/Consensus/Node/ProtocolInfo.hs

```diff
@@ -30,4 +30,5 @@ protocolInfo nodes nid demoProtocol = case demoProtocol of
     ProtocolMockPraos      params          -> protocolInfoPraos     nodes nid params
     ProtocolLeaderSchedule params schedule -> protocolInfoPraosRule nodes nid params schedule
     ProtocolMockPBFT       params          -> protocolInfoMockPBFT  nodes nid params
-    ProtocolRealPBFT       params gc mplc  -> protocolInfoByron     nodes nid params gc mplc
+    ProtocolRealPBFT       gc mplc mthr ->
```

Comment:
Just keep it on one line, like the others above it.

Reply:
Done!